web/src/pages/project/SeqrSync.css

```diff
@@ -0,0 +1,11 @@
+.checkbox-dropdown-container {
```

Comment:
Would you consider adding this inline on the component - rather than creating a new styles.css.

Reply:
Yeah I like it better with inline too

![image](https://github.com/populationgenomics/metamist/assets/34049565/8d973072-5d6d-4913-bdf0-65a57751525e)